src/components/cities/cities.tsx

```diff
@@ -12,20 +13,32 @@ type CitiesProps = {
 
 export default function Cities({ offers, city }: CitiesProps): JSX.Element {
   const [activeCardId, setActiveCard] = useState<ActiveCardId>(null);
```

Comment:
Можно было не заводить отдельный тип под это, плюс так я не вижу сразу что тип может быть null без того чтобы провалиться в ActiveCardId. Гораздо нагляднее будет написать
```suggestion
  const [activeCardId, setActiveCard] = useState<Offer['id'] | null>(null);
```